learn/glossary.md

```diff
@@ -53,18 +93,34 @@ A **Relay Response** is broken down into 2 sections:
 * Response Payload
 * Servicer Signature
 
-Requests are signed by the Client of the Application and responses are signed by the servicing Validator.
+Requests are signed by the client of the application and responses are signed by the servicing validator.
 
-This 1 for 1 signature scheme enables the protocol to validate all parties that participate in the _servicing_ cycle.
+This one-to-one signature scheme enables the protocol to validate all parties that participate in the servicing cycle.
 
 ## Relay Chain
 
-A single database instance, typically a blockchain, provided by a Validator that makes up the Pocket Network decentralized infrastructure.
+Single database instance, typically a blockchain, provided by a Validator that makes up the Pocket Network decentralized infrastructure.
 
 ## Relay Evidence
 
 Provable evidence of a Relay completed, backed by digital signatures from an Application client.
 
+## RelaysToTokensMultiplier
+
+[On-chain parameter](protocol-parameters.md#relaystotokensmultiplier) that signifies the amount of POKT that is awarded to a node for a single successful relay. This value is periodically adjusted in line with Pocket Network' [monetary policy](economics/monetary-policy.md).
+
+## Remote Procedure Call (RPC)
+
+Standardized process where one computer on a network sends a synchronous request to another. An advantage to an RPC is that the client issuing the request need not know much about the server. Many blockchains communicate by means of RPC. An RPC server may either be public or private.
+
+## Secret Key
+
+Security feature for applications created in the Pocket Portal. If "Private Secret Key Required" is selected in the Pocket Portal, then the Secret Key will need to be sent along with the request using HTTP Basic Authentication.
+
+## Servicer
+
+Any node which is eligible to serve RPC requests from applications. By default, a node is a Servicer.
+
 ## ServiceURI
 
 The endpoint where Validators host the Pocket RPC.
```

Comment:
The endpoint that Servicers expose to provide Applications the functionality they need to send relays via the Pocket Network RPC to said Servicer.